src/test/java/org/jenkinsci/plugins/durabletask/PowershellScriptTest.java

```diff
@@ -161,13 +161,55 @@
     
     @Test public void verbose() throws Exception {
         DurableTask task = new PowershellScript("$VerbosePreference = \"Continue\"; Write-Verbose \"Hello, World!\"");
+        Controller c = task.launch(new EnvVars(), ws, launcher, listener);
+        while (c.exitStatus(ws, launcher, listener) == null) {
+            Thread.sleep(100);
+        }
+        ByteArrayOutputStream baos = new ByteArrayOutputStream();
+        c.writeLog(ws, baos);
+        assertEquals(0, c.exitStatus(ws, launcher).intValue());
+        assertThat(baos.toString(), containsString("VERBOSE: Hello, World!"));
+        c.cleanup(ws);
+    }
+    
+    @Test public void verboseNegativeTest() throws Exception {
```

Comment:
how is this a negative test (negative tests generally check for error handling)?

Reply:
That's right.  This is poorly named.  I am trying to ensure that when using returnStdout another PowerShell output stream does not get mixed in.  I'll rename the test.